packages/simple_trail/Gemfile

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+# Declare your gem's dependencies in simple_trail.gemspec.
+# Bundler will treat runtime dependencies like base dependencies, and
+# development dependencies will be added by default to the :development group.
+gemspec
+
+rails_version = ENV.fetch("RAILS_VERSION", ">= 5")
```

Comment:
it looks like we are only testing Rails 6 versions, so we should probably specify that here.